src/client/method-default-target.c

```diff
@@ -105,23 +100,37 @@ static int method_set_default_target_on(Client *client, char *node_name, char *t
 
 void usage_method_get_default_target() {
         usage_print_header();
-        usage_print_usage("bluechictl get-default [nodename]");
+        usage_print_usage("bluechictl get-default [nodename] [options]");
+        printf("\n");
+        printf("Available options:\n");
+        printf("  --%s \t shows this help message\n", ARG_HELP);
 }
 
 void usage_method_set_default_target() {
         usage_print_header();
-        usage_print_usage("bluechictl set-default [nodename] [target]");
+        usage_print_usage("bluechictl set-default [nodename] [target] [options]");
+        printf("\n");
+        printf("Available options:\n");
+        printf("  --%s \t shows this help message\n", ARG_HELP);
+        printf("  --%s \t if set, forces the new default target \n", ARG_FORCE);
```

Comment:
Related to above comment about systemctl implementation

Reply:
Removed :heavy_check_mark: